optaplanner-wb-webapp/pom.xml

```diff
@@ -546,6 +546,26 @@
       <artifactId>kie-wb-common-library-backend</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-backend</artifactId>
+    </dependency>
+
+    <!-- KIE Workbench Playground Repository -->
+    <dependency>
+      <groupId>org.kie.workbench.playground</groupId>
+      <artifactId>playground-parent</artifactId>
```

Comment:
@ge0ffrey I agree the structure could be reworked to make things a bit more transparent. I created https://issues.jboss.org/browse/GUVNOR-2862 to fix this.